docs/templating/README.md

```diff
@@ -11,9 +11,9 @@ In the overview below, if you start with a page on the left and trace it to the
 
 ## Template Hierarchy in Next.js
 
-While Next.js has a robust [routing layer](https://nextjs.org/docs/routing/introduction), it is not always ideal for WordPress-based sites as the routes are built around directory and file structures rather than content types.
+While Next.js has a robust [routing layer](https://nextjs.org/docs/routing/introduction), it is not always ideal for WordPress-based sites routes are built around directory and file structures rather than content types.
```

Comment:
```suggestion
While Next.js has a robust [routing layer](https://nextjs.org/docs/routing/introduction), it is not always ideal for WordPress-based sites where routes are built around directory and file structures rather than content types.
```